spacecmd/spacecmd.changes

```diff
@@ -1,3 +1,4 @@
+- Validate channel doesn't have cloned channels before delete it(bsc#1138454)
```

Comment:
"Validate that"
"before deleting it (bsc#1138454)"

Reply:
done. Thanks